pkg/apis/batch/v1alpha1/job.go

```diff
@@ -72,6 +72,10 @@ type JobSpec struct {
 	// +optional
 	Plugins map[string][]string `json:"plugins,omitempty" protobuf:"bytes,6,opt,name=plugins"`
 
+	// Job Running Estimate is the running estimate for job
+	// Default to nil (take action immediately).
+	JobRunningEstimate *metav1.Duration `json:"jobRunningEstimate,omitempty" protobuf:"bytes,4,opt,name=jobRunningEstimate"`
```

Comment:
please update the index, it is unique

Reply:
Job prefix seems duplicate and suggest `EstimateRunningTime` or some other names to make it clear